src/schemas/21-686C/schema.js

```diff
@@ -74,12 +74,19 @@ const commonAddressFields = {
   }
 }
 
+// 4494 NOTE: It isn't immediately clear to me why these definitions are used for marriage instead
+// of the ones found in ../common/definitions.js. This will require additional investigation. 
+// For now, I've added marriageType directly to the defintion below.
+
 const commonMarriageDef = {
-  required: ['dateOfMarriage', 'locationOfMarriage', 'spouseFullName'],
+  required: ['dateOfMarriage', 'locationOfMarriage', 'spouseFullName', 'marriageType'],
   properties: {
     dateOfMarriage: schemaHelpers.getDefinition('date'),
     locationOfMarriage: {
-      $ref: '#/definitions/location'
+      $ref: '#/definitions/genericLocation'
+    },
+    marriageType: {
+      type: 'string'
```

Comment:
Similar to the pensions form. This is a field required on the paper 686, and needed to be added here.